package.json

```diff
@@ -47,6 +48,7 @@
     "eslint-plugin-prettier": "2.3.1",
     "eslint-plugin-react": "7.7.0",
     "eslint-plugin-react-native-animation-linter": "0.1.1",
+    "eslint-watch": "^4.0.1",
```

Comment:
This was super helpful when making this change b/c vscode only shows lint issues in the opened files where as this will show lint errors in all files.